src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/StomperBoots.java

```diff
@@ -47,7 +47,12 @@ public void stomp(EntityDamageEvent fallDamageEvent) {
         p.setVelocity(new Vector(0, 0.7, 0));
 
         for (Entity n : p.getNearbyEntities(4, 4, 4)) {
-            if (n instanceof LivingEntity && n.isValid() && !n.getUniqueId().equals(p.getUniqueId())) {
+            if (n instanceof LivingEntity
+                && n.isValid()
+                && !n.getUniqueId().equals(p.getUniqueId())
+                && ((LivingEntity) n).isCollidable()
+                && n.hasGravity()
+            ) {
```

Comment:
I think this unholy abomination of new lines justifies to have its own dedicated method.

Reply:
"abomination"

you seen a few lines down? 181 chars on a line monkaS

Reply:
Still, putting this statement into its own method would also allow for flexibility in case a sub implementation wants to add additional checks.